source/developer/bot-accounts.rst

```diff
@@ -3,25 +3,25 @@
 Bot Accounts
 ========================================
 
-Use Bot Accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the :doc:`personal access tokens feature <personal-access-tokens>`.
+Use Bot accounts to integrate with Mattermost through `plugins <https://developers.mattermost.com/extend/plugins/>`_ or the `Mattermost RESTful API <https://api.mattermost.com>`_. Bot accounts access the RESTful API on behalf of a bot through the use of the :doc:`personal access tokens feature <personal-access-tokens>`.
 
 Bot accounts are just like user accounts, except they:
 
-  - Cannot be logged into.
-  - Cannot be used to create other bot accounts.
-  - Do not count as a registered user and therefore do not count towards the total users for an Enterprise Edition license.
+  - Can't be logged into.
+  - Can't be used to create other bot accounts.
+  - Don't count as a registered user and therefore don't count towards the total users for an Enterprise Edition license.
 
 Additional benefits include:
 
-  - Bot accounts can be enabled to post to any channel in the system by System Administrators, including a private team, private channel or a Direct Message channel.
+  - Bot accounts can be enabled to post to any channel in the system by System Admins, including a private team, private channel or a Direct Message channel.
   - Integrations created by a user and tied to a bot account no longer break if the user leaves the company.
   - Once created, bot accounts behave just like regular user accounts and can be added to teams and channels similar to users.
   - Bot accounts are a safer way to integrate with Mattermost through the RESTful API and Plugin API because there is no need to manage shared logins with these accounts.
   - ``BOT`` tag is used everywhere in the UI where bot accounts are referenced, including messages and user lists.
 
 Note that currently:
 
-  - Bot accounts can only be created or managed by plugins or System Administrators.
+  - Bot accounts can only be created or managed by plugins or System Admins.
   - Bot accounts cannot be assigned to webhooks or slash commands. These must still be created by a user account.
```

Comment:
On initial read, these two sentences seem to be describing different functions. The first sentence is about assigning bot accounts to webhooks or slash commands, and the second sentence is about creating webhooks or slash commands.